macvmio/curie
# Add `rmi` command

`rm` command can only remove containers now. Added new command `rmi` to remove images.

Test Plan:
- Ensure all CI checks pass
- Verify both `curie rm` can delete containers
- Verify both `curie rmi` can delete images
